integration/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.tutrit</groupId>
+        <artifactId>web-client</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>integration</artifactId>
+
+    <properties>
+        <exclude.tests>nothing-to-exclude</exclude.tests>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
We've talked to have only one implementation dependency, to avoid bean conflicts. leave only one

Reply:
I did